### DISCUSSION TITLE
Philips TV Integration, cannot change input

### DISCUSSION THREAD

**Discussion by Pawllos101** [community/community#119967] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,
I have Philips PUS 6561 which I integrated with HA via Philips integration. I can change volume, turn off tv but I cannot change input to different HDMI. I tried by picking hdmi, tried media_player.select_source and nothing works. Is there any way to make it work?
Thanks for help.
Pawllos101

**Comment by ebndev** [comment]
Hi @Pawllos101,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
